lms/djangoapps/courseware/views/views.py

```diff
@@ -523,6 +528,20 @@ def render_to_standalone_html(self, request, fragment, course=None, tab=None, **
             },
         )
 
+    @staticmethod
+    def add_new_relic_course_key_metrics(course_key):
+        """
+        Initialize metrics (course_id and org) for New Relic from a course_key,
```

Comment:
Nit: docstrings should start with a single line summary:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings